src/message/message.rs

```diff
@@ -63,6 +64,14 @@ pub struct Message {
     #[serde(skip_serializing_if = "Option::is_none")]
     pub expires_time: Option<u64>,
 
+    /// from_prior is used for DID rotation
+    #[serde(skip_serializing, skip_deserializing)]
+    pub from_prior: Option<FromPrior>,
```

Comment:
What if we just deserialised the message? It will be inconsistent with from_prior_jwt and you will not be able to pack it.

Reply:
Separated packing `FromPrior` from packing `Message`. Now `Message` contain only `from_prior` field of `Option<String>` type (which can contain signed JWT with from_prior value).